api/handlers_services.go

```diff
@@ -340,13 +322,16 @@ func (s server) newOrchestrator(account string) (*orchestration.Orchestrator, er
 	}
 
 	return &orchestration.Orchestrator{
-		CloudWatchLogs:   cwlService,
-		ECS:              ecsService,
-		IAM:              iamService,
-		SecretsManager:   smService,
-		ServiceDiscovery: sdService,
-		Token:            uuid.NewV4().String(),
-		Org:              s.org,
+		CloudWatchLogs:        cwlService,
+		ECS:                   ecsService,
+		IAM:                   iamService,
+		SecretsManager:        smService,
+		ServiceDiscovery:      sdService,
+		DefaultSecurityGroups: ecsService.DefaultSgs,
+		DefaultSubnets:        ecsService.DefaultSubnets,
+		DefaultPublic:         "DISABLED",
```

Comment:
Interesting to move the defaults here. Was there any specific reason why?

Reply:
I wanted to carry them around without accidentally overriding the global, it seemed more predictable to do it this way.  It feels like that orchestrator should carry around all the stuff it needs to do the orchestration.